serviceWrappers/linux/kLiveController

```diff
@@ -57,7 +57,7 @@ loadNvm() {
 
 start() {
     [ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
-    loadNvm
+    #loadNvm
```

Comment:
To solve the issue, and make everyone happy, we can keep a variable for USE_NVM or LOAD_NODE_VIA_NVM (true/false).
In production we choose to use nvm so we know that the version of node is exactly as the version that was approved by RnD, and planned to be used.
When the product started (LiveDVR) it used node 0.10.40, and ran in the same time with then new LiveController which needed Node 6.3.0, so using nvm was the way to go.
if in future versions we will not need to run different node versions on the same server, and do not fear upgrades that will change code behaviuor, we can go with official installed rpm (deb) packages.
IMHO, nvm is the way to go, as it defines in code the specific node versions, but I'm no expert on OnPrems installation standards.